File: official/docs/node/v5/addresses/verify-failure.js
const Easypost = require('@easypost/api');

const api = new Easypost('EASYPOST_API_KEY');

const address = new api.Address({
  street1: 'UNDELIVERABLE ST',
  city: 'SAN FRANCISCO',
  state: 'CA',
  zip: '94104',
  country: 'US',
  company: 'EasyPost',
  phone: '415-123-4567',
});

address.save().then(console.log);

File: official/docs/golang/v2/addresses/verify-failure.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	address, _ := client.CreateAddress(
		&easypost.Address{
			Street1: "UNDELIVERABLE ST",
			City:    "SAN FRANCISCO",
			State:   "CA",
			Zip:     "94104",
			Country: "US",
			Company: "EasyPost",
			Phone:   "415-123-4567",
		},
		&easypost.CreateAddressOptions{},
	)

	fmt.Println(address)
}
